src/styles/Css/App.css

```diff
@@ -69,6 +70,85 @@ button:focus {
   gap: 2rem;
 }
 
+.navBar {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  align-self: stretch;
+  padding: 0.5rem 0.5rem 0.2rem 1.5rem;
+  box-shadow: 5px 5px 8px rgba(0, 0, 0, 0.1), 5px 2px 8px rgba(0, 0, 0, 0.1);
+  background: #fff;
```

Comment:
- [ ] Same thing here again the background #fff has been used several times, using a reusable class will make the codebase DRY. Kindly do so